cmd/chart-repo/utils.go

```diff
@@ -310,20 +310,35 @@ func fetchAndImportIcon(dbSession datastore.Session, c chart) error {
 		return fmt.Errorf("%d %s", res.StatusCode, c.Icon)
 	}
 
-	orig, err := imaging.Decode(res.Body)
-	if err != nil {
-		return err
-	}
+	b := []byte{}
+	contentType := ""
+	if strings.Contains(res.Header.Get("Content-Type"), "image/svg") {
+		// if the icon is a SVG file simply read it
+		b, err = ioutil.ReadAll(res.Body)
+		if err != nil {
+			return err
+		}
+		contentType = res.Header.Get("Content-Type")
+	} else {
+		// if the icon is in any other format try to convert it to PNG
+		orig, err := imaging.Decode(res.Body)
+		if err != nil {
+			log.Infof("failing to decode %v", err)
```

Comment:
This should be an `Errorf`, also can we add fields that describe what chart this icon is for? E.g. https://github.com/helm/monocular/pull/631/files#diff-dd464ab7a9989bed9c6a3801d51ade13R288

```suggestion
			log.WithFields(log.Fields{"name": c.Name}).WithError(err).Error("failed to decode icon")
```